src/magic_enchantment.cpp

```diff
@@ -181,6 +181,31 @@ bool enchantment::is_active( const Character &guy, const item &parent ) const
     return false;
 }
 
+// TODO: add move mutation ref and checks here
+bool enchantment::is_active(const Character &guy) const
```

Comment:
i recommend calling this function in the other overload in order to reduce duplicated code.

Reply:
did as you recommended